src/modules/keyboardmanager/ui/ShortcutControl.h

```diff
@@ -22,9 +26,16 @@ class ShortcutControl
     static KeyboardManagerState* keyboardManagerState;
     // Stores the current list of remappings
     static std::vector<std::vector<Shortcut>> shortcutRemapBuffer;
+    // Vector to store dynamically allocated KeyDropDownControl objects to avoid early destruction
+    std::vector<std::unique_ptr<KeyDropDownControl>> keyDropDownControlObjects;
 
-    ShortcutControl(const int& rowIndex, const int& colIndex)
+    ShortcutControl(const size_t rowIndex, const size_t colIndex)

```

Comment:
Why `const size_t` here? Seems unnecesary.

Reply:
It's `size_t` since I'm passing in a `vector.size()` as argument while creating the object.